src_assets/common/assets/web/index.html

```diff
@@ -107,21 +107,27 @@ <h2>Legal</h2>
         let v = this.githubVersion.name;
         // If the version starts with a v, remove it
         if (v.indexOf("v") === 0) v = v.substring(1);
+
+        // if nightly, we do an additional check to make sure its an actual upgrade.
+        if (this.version.split('.').length > 3) {
+          const stableVersion = this.version.split('.').slice(0, 3).join('.');
+          return this.githubVersion.tag_name.substring(1) > stableVersion;
+        }
+
         // return true if the version is different, otherwise false
         return v !== this.version.split(".")[0];
       },
-      // Check for new nightly version
```

Comment:
Besides the fact the comment is obvious, is there another reason it was removed?

Reply:
That was the main reason, it was a bit too obvious. It’s often suggested in clean code to remove obvious comments at times.